### DISCUSSION TITLE
Exception message: The Internet connection appears to be offline

### DISCUSSION THREAD

**Discussion by travisboss** [dani-garcia/vaultwarden#1937] [Q&A] [answered]
### Subject of the issue

Trying to login to my local hosted environment on my iphone 12 pro max on ios 14.7.1, when inputting my server location *****.duckdns.org and proceeding to input my username and password I get the error above `Exception message: The Internet connection appears to be offline` if I try again the app simply closes on me with no error, in checking my logs I do not see a connection being made `sudo docker logs -f bitwarden`

### Deployment environment
OpenMediaVault using docker and upgraded to `vaultwarden/server:latest`

* vaultwarden version: 1.22.1

* Install method: Docker image

* Clients used:  iOS

* Reverse proxy and version:  using linuxserver/swag

* MySQL/MariaDB or PostgreSQL version:  Using what is built in to vaultwarden, do I need to change this, cannot seem to find a guide on using my own MariaDB(which I have setup already for nextcloud).

* Other relevant details:

### Steps to reproduce

Trying to login to my local hosted environment on my iphone 12 pro max on ios 14.7.1, when inputting my server location *****.duckdns.org and proceeding to input my username and password I get the error above `Exception message: The Internet connection appears to be offline` if I try again the app simply closes on me with no error, in checking my logs I do not see a connection being made `sudo docker logs -f bitwarden`

### Expected behaviour

Login to vaultwarden

### Actual behaviour

Application closes

**Comment by BlackDex** [comment]
If you do not see a connection being made, then it doesn't seem to be a Vaultwarden issue. What happens if you try to connect to the default vault.bitwarden.com ?

And, do you also see nothing in the swag/reverse proxy logs?

**Reply by travisboss** [reply]
I am able to connect fine with a the standard vault.bitwarden.com

I do not see anything in the logs.

As mentioned this is the only device this is not working on all other computers and my wifes ios device work fine on connecting.

**Comment by travisboss** [comment, marked as answer]
Figured it out, I for some reason had authelia on in bitwarden swag config and the app could not figure this out, once disabling authelia it works fine now.
